.github/workflows/github-token-rotation-prod.yml

```
@@ -0,0 +1,96 @@
+          role-to-assume: ${{ secrets.AWS_GITHUB_TOKEN_ROTATION_ROLE_TO_ASSUME_PROD }}
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Verify AWS credentials
+        run: |
+          if ! aws sts get-caller-identity &>/dev/null; then
+            echo "Error: Failed to configure AWS credentials. Please check the role configuration."
+            exit 1
+          fi
+
+      - name: Setup Python and Enable Pip Cache
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)